manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -117,7 +117,7 @@ type DataFlow struct {
 	Name string `json:"name"`
 
 	// +required
-	// +kubebuilder::validation:MinItems:=1
+	// +kubebuilder:validation:MinItems:=1
```

Comment:
Does it really make sense to add annotations in a resource that we auto generate? I mean it doesn't check user input but rather that our codebase has no bug and I'm not sure if that's the best way to check it (it will just keep rescheduling a reconcile)

Reply:
I remember that we decided to add the validation for developer purposes (to create a blueprint and get around all the opa/catalog configurations). I can remove the check, though.